src/Components/CreateImageWizard/steps/oscap.js

```diff
@@ -20,8 +26,9 @@ const oscapStep = {
       name: 'oscap-text-component',
       label: (
         <Text>
-          Monitor regulatory compliance profiles of registered RHEL systems you
-          must adhere to via OpenSCAP.
+          Use OpenSCAP to monitor the adherence of your registered RHEL systems
+          to a selected regulatory compliance profile.
+          <DocumentationButton />
```

Comment:
The documentation buttons needs to be on the same line as the text if we want to keep the space between the button and the text:

Same line:
![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/00a719ea-b5db-451d-8674-3900760010dd)

Separate line: 
![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/8015ea1f-8e33-42cf-8123-c4e82022c4ed)


```suggestion
          Use OpenSCAP to monitor the adherence of your registered RHEL systems
          to a selected regulatory compliance profile. <DocumentationButton />
```